hazelcast/src/test/java/com/hazelcast/topic/TopicStressTest.java

```diff
@@ -56,6 +56,7 @@
     public void setUp() {
         TopicConfig topicConfig = new TopicConfig();
         topicConfig.setName("topic*");
+        topicConfig.setMultiThreadingEnabled(true);
```

Comment:
You can do this test parameterized on `multiThreadingEnabled` by setting it to `true/false` in each run as in [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/map/nearcache/NearCacheTest.java#L82)